src/pages/Models/Details/generators.js

```diff
@@ -162,15 +162,15 @@ export function generateApplicationRows(
           content: scale,
           className: "u-align--right",
         },
-        {
-          "data-test-column": "store",
-          content: store,
-        },
         {
           "data-test-column": "revision",
           content: rev,
           className: "u-align--right",
         },
+        {
```

Comment:
The header was moved but the content was not :)

![Screen Shot 2020-11-27 at 9 51 25 AM](https://user-images.githubusercontent.com/532033/100466220-2f328600-3096-11eb-8c81-42c210635c80.png)

Reply:
Good catch.. fixed